WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -372,6 +366,7 @@ class PostListViewModel @Inject constructor(
         }
     }
 
+    // TODO: Can this be a private method?
```

Comment:
I think we can just make it private as part of this PR. It's such a small change, that it doesn't really matter - I believe we are safe as long as the build succeeds.